content/post/22-03-16-progress-report-31.md

```diff
@@ -0,0 +1,165 @@
++++
+title = "Progress Report #31: Duck, Duck, Weh"
+author = "ike709"
+date = 2022-03-16
+categories = [
+    "Progress Report"
+]
+reddit = ""
+thumbnail = "pr_31"
++++
+It's been a minute since the last Progress Report, and there's been so many changes that we had to break this one up into two parts! This part covers January 2022.
+<!--more-->
+
+{{< newsection >}}
+## Content
+{{< newsection >}}
+### Two-Parter
+There's been a long list of contributions since the last progress report, so this edition just covers January 2022. The February 2022 edition will follow in a week or so.
+
+{{< newsection >}}
+### Game Wiki
+
+Space Station 14 now has a [wiki](https://wiki.spacestation14.io/wiki/Main_Page)! It's a one-stop shop for everything you need to know in order to get that greentext.
```

Comment:
Perhaps mention that it is open to contributions? People can log in with their SS14 account.